crates/sui-json-rpc-types/src/sui_move.rs

```diff
@@ -501,6 +501,14 @@ impl SuiMoveStruct {
             }
         }
     }
+
+    pub fn read_dynamic_field_value(&self, field_name: &str) -> Option<SuiMoveValue> {
```

Comment:
Let's rename this to something like `field_value` or `field_by_name` or something similar?